plugins/chat/lib/chat_notifier.rb

```diff
@@ -60,6 +60,7 @@ def initialize(chat_message, timestamp)
     @timestamp = timestamp
     @chat_channel = @chat_message.chat_channel
     @user = @chat_message.user
+    @mentions = Chat::ChatMessageMentions.new(chat_message)
```

Comment:
It's a nitpick, but I would generally not expect an initializer to have so much side effects right away (here, SQL queries).

I would expect it more from an API similar to:

```ruby
@mentions = Chat::ChatMessageMentions.new(chat_message)
@mentions.parse
```

or 

```ruby
@mentions = Chat::ChatMessageMentions.parse(chat_message)
```